### PR TITLE
Fix "using Qt [version]" message being too far down

### PR DESCRIPTION
**Pull request by iamthesenate1**
Before:
screenshot-80bcbdc2

After:
screenshot-459cee6a

### CODE REVIEW

**Comment by iamthesenate1** [comment]
Another improvement is making the frame border invisible:
Before:
screenshot-459cee6a

After:
screenshot-2d1bda49

I'm not sure if this should be its own PR. What do you think?

**Comment by tsujan** [comment]
I don't think there's any issue to fix but if you really want to remove that extra vertical spacing:

 1. Remove `labVersion` because it isn't used anywhere;
 2. Change the vertical alignment of `labAppName` to "AlignBottom"; and
 3. Change the vertical alignment of `labQtVer` to "AlignTop".

Nothing else is needed.

Particularly, never use a specific font or font size in the ui file.

As for the frame border, please don't change it. These things are about taste, and tastes differ.

**Comment by iamthesenate1** [comment]
> I don't think there's any issue to fix but if you really want to remove that extra vertical spacing:
> 
>     1. Remove `labVersion` because it isn't used anywhere;
> 
>     2. Change the vertical alignment of `labAppName` to "AlignBottom"; and
> 
>     3. Change the vertical alignment of `labQtVer` to "AlignTop".
> 
> 

Done.

> Particularly, never use a specific font or font size in the ui file.
> 
> As for the frame border, please don't change it. These things are about taste, and tastes differ.

Ok. I understand.

File: src/core/ui/aboutwidget.ui [hunk 1]
```diff
@@ -64,17 +64,7 @@
...
-        </widget>
-       </item>
-       <item>
-        <widget class="QLabel" name="labVersion">
-         <property name="text">
-          <string notr="true"/>
-         </property>
-         <property name="alignment">
-          <set>Qt::AlignCenter</set>
+          <set>Qt::AlignBottom</set>
```
Comment (tsujan) [inline src/core/ui/aboutwidget.ui:67]: It should be `Qt::AlignBottom|Qt::AlignHCenter`

File: src/core/ui/aboutwidget.ui [hunk 2]
```diff
@@ -84,7 +74,7 @@
           <string notr="true"/>
          </property>
          <property name="alignment">
-          <set>Qt::AlignCenter</set>
+          <set>Qt::AlignTop</set>
```
Comment (tsujan) [inline src/core/ui/aboutwidget.ui:77]: It should be `Qt::AlignHCenter|Qt::AlignTop`.

**Review by tsujan — CHANGES_REQUESTED** [review]

**Comment by tsujan** [comment]
Now it's good.

**Review by tsujan — APPROVED** [review]

**Comment by iamthesenate1** [comment]
Ok. Thank you very much.

**Comment by tsujan** [comment]
Thank you.

**Pull request merged**